// @esri/calcite-components/dist/cjs/id-d01bcb1a.js
/*!
 * All material copyright ESRI, All Rights Reserved, unless otherwise specified.
 * See https://github.com/Esri/calcite-design-system/blob/dev/LICENSE.md for details.
 * v2.13.0
 */
'use strict';

const calciteInputTimePicker_calciteTimePicker_entry = require('./calcite-input-time-picker.calcite-time-picker-d7f98bc8.js');
require('./index-03561b9a.js');
require('./form-26bde25c.js');
require('./dom-401c47fa.js');
require('./guid-7487a6a8.js');
require('./resources-8d0a3a55.js');
require('./interactive-e27cd6e3.js');
require('./key-8032a9dc.js');
require('./label-4cec1587.js');
require('./component-25eaec79.js');
require('./loadable-1e343f4a.js');
require('./browser-b9024aa0.js');
require('./locale-783be19a.js');
require('./observers-dc37fb24.js');
require('./focusTrapComponent-f0b8db52.js');
require('./config-86d6376a.js');
require('./math-07ce0aa7.js');
require('./t9n-5f0809e3.js');
require('./Validation-64dfab90.js');
require('./input-3dd5805a.js');

// Indonesian [id]
var locale = {
  name: 'id',
  weekdays: 'Minggu_Senin_Selasa_Rabu_Kamis_Jumat_Sabtu'.split('_'),
  months: 'Januari_Februari_Maret_April_Mei_Juni_Juli_Agustus_September_Oktober_November_Desember'.split('_'),
  weekdaysShort: 'Min_Sen_Sel_Rab_Kam_Jum_Sab'.split('_'),
  monthsShort: 'Jan_Feb_Mar_Apr_Mei_Jun_Jul_Agt_Sep_Okt_Nov_Des'.split('_'),
  weekdaysMin: 'Mg_Sn_Sl_Rb_Km_Jm_Sb'.split('_'),
  weekStart: 1,
  formats: {
    LT: 'HH.mm',
    LTS: 'HH.mm.ss',
    L: 'DD/MM/YYYY',
    LL: 'D MMMM YYYY',
    LLL: 'D MMMM YYYY [pukul] HH.mm',
    LLLL: 'dddd, D MMMM YYYY [pukul] HH.mm'
  },
  relativeTime: {
    future: 'dalam %s',
    past: '%s yang lalu',
    s: 'beberapa detik',
    m: 'semenit',
    mm: '%d menit',
    h: 'sejam',
    hh: '%d jam',
    d: 'sehari',
    dd: '%d hari',
    M: 'sebulan',
    MM: '%d bulan',
    y: 'setahun',
    yy: '%d tahun'
  },
  ordinal: function ordinal(n) {
    return n + ".";
  }
};
calciteInputTimePicker_calciteTimePicker_entry.dayjs.locale(locale, null, true);

exports['default'] = locale;
