// @esri/calcite-components/dist/cjs/mk-8a9bf302.js
/*!
 * All material copyright ESRI, All Rights Reserved, unless otherwise specified.
 * See https://github.com/Esri/calcite-design-system/blob/dev/LICENSE.md for details.
 * v2.13.0
 */
'use strict';

const calciteInputTimePicker_calciteTimePicker_entry = require('./calcite-input-time-picker.calcite-time-picker-d7f98bc8.js');
require('./index-03561b9a.js');
require('./form-26bde25c.js');
require('./dom-401c47fa.js');
require('./guid-7487a6a8.js');
require('./resources-8d0a3a55.js');
require('./interactive-e27cd6e3.js');
require('./key-8032a9dc.js');
require('./label-4cec1587.js');
require('./component-25eaec79.js');
require('./loadable-1e343f4a.js');
require('./browser-b9024aa0.js');
require('./locale-783be19a.js');
require('./observers-dc37fb24.js');
require('./focusTrapComponent-f0b8db52.js');
require('./config-86d6376a.js');
require('./math-07ce0aa7.js');
require('./t9n-5f0809e3.js');
require('./Validation-64dfab90.js');
require('./input-3dd5805a.js');

// Macedonian [mk]
var locale = {
  name: 'mk',
  weekdays: 'недела_понеделник_вторник_среда_четврток_петок_сабота'.split('_'),
  months: 'јануари_февруари_март_април_мај_јуни_јули_август_септември_октомври_ноември_декември'.split('_'),
  weekStart: 1,
  weekdaysShort: 'нед_пон_вто_сре_чет_пет_саб'.split('_'),
  monthsShort: 'јан_фев_мар_апр_мај_јун_јул_авг_сеп_окт_ное_дек'.split('_'),
  weekdaysMin: 'нe_пo_вт_ср_че_пе_сa'.split('_'),
  ordinal: function ordinal(n) {
    return n;
  },
  formats: {
    LT: 'H:mm',
    LTS: 'H:mm:ss',
    L: 'D.MM.YYYY',
    LL: 'D MMMM YYYY',
    LLL: 'D MMMM YYYY H:mm',
    LLLL: 'dddd, D MMMM YYYY H:mm'
  },
  relativeTime: {
    future: 'после %s',
    past: 'пред %s',
    s: 'неколку секунди',
    m: 'минута',
    mm: '%d минути',
    h: 'час',
    hh: '%d часа',
    d: 'ден',
    dd: '%d дена',
    M: 'месец',
    MM: '%d месеци',
    y: 'година',
    yy: '%d години'
  }
};
calciteInputTimePicker_calciteTimePicker_entry.dayjs.locale(locale, null, true);

exports['default'] = locale;
