// @esri/calcite-components/dist/esm/th-66fe507b.js
/*!
 * All material copyright ESRI, All Rights Reserved, unless otherwise specified.
 * See https://github.com/Esri/calcite-design-system/blob/dev/LICENSE.md for details.
 * v2.13.0
 */
import { d as dayjs } from './calcite-input-time-picker.calcite-time-picker-ed85a5a2.js';
import './index-b39237b3.js';
import './form-57a6688d.js';
import './dom-a412467a.js';
import './guid-8d29c18d.js';
import './resources-d219e37c.js';
import './interactive-aa12fe2b.js';
import './key-f71ed5cc.js';
import './label-d349ffcf.js';
import './component-b5b1c0d4.js';
import './loadable-c5106ddb.js';
import './browser-0dbc8b58.js';
import './locale-cc91d254.js';
import './observers-d7a56c1f.js';
import './focusTrapComponent-61b41d5a.js';
import './config-64ce2a17.js';
import './math-470fa9f6.js';
import './t9n-861497d5.js';
import './Validation-28f956cd.js';
import './input-e52dc73d.js';

// Thai [th]
var locale = {
  name: 'th',
  weekdays: 'อาทิตย์_จันทร์_อังคาร_พุธ_พฤหัสบดี_ศุกร์_เสาร์'.split('_'),
  weekdaysShort: 'อาทิตย์_จันทร์_อังคาร_พุธ_พฤหัส_ศุกร์_เสาร์'.split('_'),
  weekdaysMin: 'อา._จ._อ._พ._พฤ._ศ._ส.'.split('_'),
  months: 'มกราคม_กุมภาพันธ์_มีนาคม_เมษายน_พฤษภาคม_มิถุนายน_กรกฎาคม_สิงหาคม_กันยายน_ตุลาคม_พฤศจิกายน_ธันวาคม'.split('_'),
  monthsShort: 'ม.ค._ก.พ._มี.ค._เม.ย._พ.ค._มิ.ย._ก.ค._ส.ค._ก.ย._ต.ค._พ.ย._ธ.ค.'.split('_'),
  formats: {
    LT: 'H:mm',
    LTS: 'H:mm:ss',
    L: 'DD/MM/YYYY',
    LL: 'D MMMM YYYY',
    LLL: 'D MMMM YYYY เวลา H:mm',
    LLLL: 'วันddddที่ D MMMM YYYY เวลา H:mm'
  },
  relativeTime: {
    future: 'อีก %s',
    past: '%sที่แล้ว',
    s: 'ไม่กี่วินาที',
    m: '1 นาที',
    mm: '%d นาที',
    h: '1 ชั่วโมง',
    hh: '%d ชั่วโมง',
    d: '1 วัน',
    dd: '%d วัน',
    M: '1 เดือน',
    MM: '%d เดือน',
    y: '1 ปี',
    yy: '%d ปี'
  },
  ordinal: function ordinal(n) {
    return n + ".";
  }
};
dayjs.locale(locale, null, true);

export { locale as default };
